### DISCUSSION TITLE
Current balance is consistantly incorrect

### DISCUSSION THREAD

**Discussion by nottheend** [firefly-iii/firefly-iii#6896] [General]
I am regularly reconciling my account according to my bank statement. Everything looks fine.
However, look a the "current balance" on the homepage, it is somehow completely off (e.g. by 1300).

Do you have any idea how to get closer to the issue?

I tried to call `/flush`, but nothing changed

**Comment by JC5** [comment]
The balance on the front page. You mean in the box? That denotes the total income in the current period.

**Reply by nottheend** [reply]
Yes, I mean the balance in the box on the front page which is a dropdown button with 'Edit', 'Show', 'Reconcile'...
But the same amount is also shown on the 'accounts' overview page.

I don't have any further incomes entered for this month, also no automated transaction.

Also virtual balance of that account is 0. I can't understand why after reconciling completely in the beginning of the month it is off by more than 1000

**Reply by JC5** [reply]
Mmm, I'm not really sure. There's not much to it, math-wise. You could try to see if the audit report shows you anything strange.

**Reply by nottheend** [reply]
I have counted 1.5 weeks ago and entered all transactions. Now I have again 7 off in my Cash Wallet. Firefly-III shows me more than I have actually.
Wondering if I am missing something substantially.
Is there a way to trace the calculation?

**Reply by JC5** [reply]
The audit report under `/reports` has a running balance column.

**Reply by nottheend** [reply]
Okay, thanks. After tracking my wallet and account much more closely, my guess became much stronger that I am missing a transaction here and there. I am missing 70 on my account again 🔢 
I also looked at the data importer and like the idea, but I am afraid that I lose more control ultimately. It would be cool if the imported data could be matched with the already entered transactions and show differences or set the 'clear' flag if it is fine
